### DISCUSSION TITLE
Timer like a Start-Sleep in PowerShell

### DISCUSSION THREAD

**Discussion by ricoroodenburg** [postmanlabs/postman-flows#346] [Ideas]
Hi,

It should be nice if you can add a timer step within a sequence.
For example "Run this step after some seconds".

Please make it also possible to automatic calculate the seconds, based on input time vs now.
So you can run a step after some specific time, for example after 5:00.

This way you can schedule some actions in the sequence.

**Comment by samay-v** [comment]
You can achieve the "Run this step after some seconds" use case with the `Delay` block. 
We'l try to make delay smart enough to recognise time as an input as well
